package.json

```diff
@@ -87,6 +87,16 @@
       {
         "command": "mesonbuild.restartLanguageServer",
         "title": "Meson: Restart Language Server"
+      },
+      {
+        "command": "mesonbuild.node.runAll",
```

Comment:
It feels like we should mention tests in the title and the command. Should we add separate commands for benchmarks?

Reply:
That command is only attached to a test (or benchmark) node as an inline button and context menu entry, so making the title "Run all tests" is a little redundant IMHO. Plus like you say, then requires separate commands for benchmarks.

Reply:
Do you not see value in separating tests and benchmarks?

Reply:
I haven't merged the nodes together (that would be doubly silly as I separated them a few years ago...). This `runAll` command just executes "on" either the tests node or the benchmarks node.

![b](https://github.com/mesonbuild/vscode-meson/assets/9592489/d1de9cf1-a098-4d60-a602-eb422fdfb5f3)